WebAPI/Controllers/CategoriesController.cs

```diff
@@ -22,4 +22,25 @@ public async Task<ActionResult<List<CategoryDetailDto>>> GetAll()
 
         return Ok(result);
     }
+
+    [HttpGet]
+    public async Task<ActionResult<string>> GetImage(Guid id)
+    {
+        var result = await _categoryService.GetByIdAsync(id);
+
+        return Ok(result.ImagePath);
```

Comment:
When you use `ActionResult<T>` then you don't need to wrap your response with `Ok` since `ActionResult` represents `OkObjectResult` by default, so, you can easily use it like:
```suggestion
        public async Task<ActionResult<string>> GetImage(Guid id) =>
                  await _categoryService.GetByIdAsync(id);
```

Same case for `ProductsController`